src/AstroImages.jl

```diff
@@ -299,7 +299,7 @@ function AstroImage(
     dims::Union{Tuple,NamedTuple}=(),
     refdims::Union{Tuple,NamedTuple}=(),
     header::FITSHeader=emptyheader(),
-    wcs::Union{WCSTransform,Nothing}=nothing;
+    wcs::Union{Vector{WCSTransform},Nothing}=nothing;
```

Comment:
What's the significance of this change? Also the docstring says it takes a `WCSTransform` but I only see methods for `Vector{WCSTransform}`, am I missing something?